src/libs/actions/BankAccounts.js

```diff
@@ -816,6 +816,27 @@ function updateReimbursementAccountDraft(bankAccountData) {
     Onyx.merge(ONYXKEYS.REIMBURSEMENT_ACCOUNT_DRAFT, bankAccountData);
 }
 
+/**
+ * Checks the given number is a valid US Routing Number
```

Comment:
Can we add the validation algorithm reference here?
```suggestion
 * Checks the given number is a valid US Routing Number
 * using ABA routingNumber checksum algorithm: http://www.brainjar.com/js/validation/
```